izpack-panel/src/main/java/com/izforge/izpack/panels/path/PathInputConsolePanel.java

```diff
@@ -42,13 +43,30 @@ public class PathInputConsolePanel extends AbstractConsolePanel
      * @param installData the install data
      * @param prompt the console prompt
      */
-    public PathInputConsolePanel(PanelView<ConsolePanel> panel, InstallData installData, Prompt prompt)
+    public PathInputConsolePanel(PanelView<ConsolePanel> panel, String targetPanel, InstallData installData, Prompt prompt)
     {
         super(panel);
+        this.targetPanel = targetPanel;
         this.installData = installData;
         this.prompt = prompt;
     }
 
+    /**
+     * Helper to return a language resource string.
+     *
+     * @param subkey the search subkey in targetPanel
+     * @return the corresponding string, or {@code <targetPanel>.<subkey>} if the string is not found
+     */
+    protected String getMessage(String subkey)
```

Comment:
Would it make sense to move ```getMessage(...)``` and the ```String targetPanel``` to AbstractConsolePanel and IzPanel ?

Reply:
@akuhtz, I was also thinking on the same line but then I thought that we will have another ticket to do code refactoring. Please let me know what is your opinion.

Reply:
@hiteshbosamiya I'm fine with the change if you want to refactor it in a next step.